cmd/pinniped-proxy/Cargo.toml

```diff
@@ -15,9 +15,16 @@ anyhow = "1.0"
 base64 = "0.13"
 hyper = "0.13"
 hyper-tls = "0.4"
+kube = { version = "0.43.0", features = ["derive"] }
+kube-runtime = "0.43.0"
+k8s-openapi = { version = "0.9.0", default-features = false, features = ["v1_17"] }
```

Comment:
Which are the implications of enabling `features = ["v1_17"]`?  Does it accept a range of k8s versions?

Reply:
I assume it's just the feature set that is enabled ie., in our case, it'll be assuming the 1.17 API. I don't see that it will affect us much (a little like using the kubectl 1.17 cli) especially since we're mainly using the lib for CRDs.